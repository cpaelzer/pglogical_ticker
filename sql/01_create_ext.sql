-- Allow running regression suite with upgrade paths
\set v `echo ${FROMVERSION:-1.3}`
SET client_min_messages = warning;
CREATE EXTENSION pglogical;
CREATE EXTENSION pglogical_ticker VERSION :'v';
