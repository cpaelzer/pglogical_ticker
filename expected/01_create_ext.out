-- Allow running regression suite with upgrade paths
\set v `echo ${FROMVERSION:-1.4}`
SET client_min_messages = warning;
CREATE EXTENSION pglogical;
CREATE EXTENSION pglogical_ticker VERSION :'v';
